update.sh

```diff
@@ -49,6 +55,37 @@ for version in "${versions[@]}"; do
 		continue
 	fi
 
+	otpMajor="${otpMajors[$rcVersion]}"
+	otpVersion="$(
+		git ls-remote --tags https://github.com/erlang/otp.git \
+			"refs/tags/OTP-$otpMajor.*"\
+			| cut -d'/' -f3- \
+			| cut -d'^' -f1 \
+			| cut -d- -f2- \
+			| sort -uV \
+			| tail -1
+	)"
+	if [ -z "$otpVersion" ]; then
+		echo >&2 "warning: failed to get Erlang/OTP version for '$version' ($fullVersion); skipping"
+		continue
+	fi
+	# TODO these aren't published anywhere (nor is the tarball we download even provided by Erlang -- it's simply a "git archive" tar provided by GitHub)...
+	otpSourceSha256="$(wget -qO- "https://github.com/erlang/otp/archive/OTP-$otpVersion.tar.gz" | sha256sum | cut -d' ' -f1)"
+
+	opensslMajor="${opensslMajors[$rcVersion]}"
+	opensslVersion="$(
+		wget -qO- 'https://www.openssl.org/source/' \
+			| grep -oE 'href="openssl-'"$opensslMajor"'[^"]+[.]tar[.]gz"' \
+			| sed -e 's/^href="openssl-//' -e 's/[.]tar[.]gz"//' \
+			| sort -uV \
+			| tail -1
+	)"
+	if [ -z "$opensslVersion" ]; then
+		echo >&2 "warning: failed to get OpenSSL version for '$version' ($fullVersion); skipping"
+		continue
+	fi
+	opensslSourceSha256="$(wget -qO- "https://www.openssl.org/source/openssl-$opensslVersion.tar.gz.sha256")"
```

Comment:
If you check the `Dockerfile`, we're using both -- if upstream publishes an explicit checksum (like OpenSSL does), we like to embed it so that we can explicitly track changes to the upstream artifacts (for example, if they discovered something was wrong with the 1.1.1a tarball they released and they simply re-uploaded, having the checksum gives us a natural cache bust in the appropriate place so we get the updated bits), and the PGP checking gives us a rough sanity check on the origin of those bits.